### ISSUE TITLE
Document best pratices for working on the repo

### ISSUE THREAD

**Issue by ghost** [manaaan/ekolivs-monorepo#4]
My stupid friend is wondering why his local package-lock.json got wonky all of a sudden. Some things from remote was removed, other things was added.

I suspect he did the things in the wrong order.

So: 

> git pull
> npm install
> (do stuff)
> npm run format
> npm run lint
> (npm run check?)
> git add, commit, push

Also:
Should we use something like pre-commit to simplify the workflow and avoid cluttering diffs with linting stuff?

**Comment by manaaan** [comment]
Yes we should just set up `pre-commit` to avoid any missed linting
